src/megolm/session_keys.rs

```diff
@@ -314,6 +314,13 @@ impl TryFrom<&str> for SessionKey {
     }
 }
 
+impl From<&ExportedSessionKey> for SessionKey {
```

Comment:
Why is this exactly needed? Why can't you use `InboundGroupSession::import()` with an `ExportedSessionKey`?

Reply:
I made these changes since in a file (`matrix-sdk-crypto/src/gossiping/machine.rs`) where I needed to create a `SessionKey`, the `vodozemac::InboundGroupSession` object was not visible. However after looking again at the available conversion functions in `matrix_sdk_crypto::olm::group_sessions::InboundGroupSession`, I noticed I can instead use the conversion function for `ForwardedMegolmV1AesSha2Content` to create the session, which uses `vodozemac::InboundGroupSession::import()` with the available `ExportedSessionKey`.

I am okay with you closing the PR then since I can make the necessary changes in https://github.com/matrix-org/matrix-rust-sdk/pull/2650. Thanks for checking this out.